### DISCUSSION TITLE
question about "options" within the imput section

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#108] [Q&A] [answered]
am taking a look at

```options is optional,
xtream_info_cache deprecated.
xtream_skip_live true or false, live section can be skipped.
xtream_skip_vod true or false, vod section can be skipped.
xtream_skip_series true or false, series section can be skipped.
```
does this only work for type: xtreme or does it work for m3u as well? and can i get a sample on how it goes

**Comment by euzu** [comment]
the options are only for xtream codes inputs.
Xtream codes api provides you access to live, vod and series seperatly. M3u api does not.
It is only possible with a xtream codes api access.

**Comment by euzu** [comment]
with api i mean the access to your iptv  provider.

**Reply by aniel300** [reply]
for my understanding xc or m3u both can give u access to live, movies and tv shows assuming they are enabled for the provider end. u are saying this project/ thoose options only works with xc and not with m3us as the imput?

**Comment by euzu** [comment]
yes because m3u playlist does not provide you the type of the  url. You need to guess what is behind a link. xtream codes  gives you a clear seperation. I think you need to understand what the difference between m3u api and xtream codes api is.

**Reply by aniel300** [reply]
for my understanding some providers follow the m3u stand which goes as follow
```
...serverurl/username/pass/chanelurl
...serverurl/movie/username/pass/movieurl
...serverurl/series/username/pass/serieurl
```
however, some provider don't follow that. in addition, i will use xc from now on, thank you

**Comment by euzu** [comment, marked as answer]
Yes xc is the better choice for input and output.

m3u-filter can  merge multiple inputs  in m3u or xc output with the known restrictions.
